worker/src/Channel/ChannelSocket.cpp

```diff
@@ -259,9 +259,9 @@ namespace Channel
 		auto* message = FBS::Message::GetMessage(msg);
 
 		// TMP: For debugging.
-		// auto s = flatbuffers::FlatBufferToString(
-		//   reinterpret_cast<uint8_t*>(msg), FBS::Message::MessageTypeTable());
-		// MS_ERROR("%s", s.c_str());
+		auto s = flatbuffers::FlatBufferToString(
+		  reinterpret_cast<uint8_t*>(msg), FBS::Message::MessageTypeTable());
+		MS_ERROR("%s", s.c_str());
```

Comment:
Can we remove this?

Reply:
I'll put it under a flag. It's intended for development and it's extremely useful.

Reply:
Please don't use `MS_ERROR`. Use `MS_DUMP`.